nunaliit2-js/src/main/js/nunaliit2/n2.widgetAnnotationEditor.js

```diff
@@ -1575,6 +1529,77 @@ POSSIBILITY OF SUCH DAMAGE.
 				}
 			});
 
+			/* Probably give this a unique ID instead of this selector */
```

Comment:
Should we give it an ID or remove the comment? Fine with leaving if you intend to come back to it.